src/react-native/11-navigation/navigation.md

```diff
@@ -2,23 +2,120 @@
 @parent learn-react-native 11
 @outline 3
 
-@description TODO
+@description Learn how to use navigation to move between screens in a React Native app.
 
 @body
 
 ## Overview
 
 In this section, you will:
 
-- TODO
+- Understand why routing is important
+- Be introduced to deep linking
+- Learn about stack-based routing
+- Understand the navigation lifecycle
+- Implement a bottom tab navigation pattern
+- Learn about the difference between stack and native stack navigation
+- Implement a stack navigation pattern
 
 ## Objective 1: Using React Native bottom tab navigation
 
-TODO
+As our application grows in complexity, more screens will be added. The React Navigation library provides a solution that allows us to move between these screens. In this section, we will cover both common navigation patterns used in both Android and iOS applications: tab navigation and stack navigation.
 
-### Concept TODO
+The first step is to install the React Navigation library. Run the following command in your terminal:
 
-TODO
+```bash
+npm install @react-navigation/native
+```
```

Comment:
We’ve been saving any actual instructions until the Setup step.

Reply:
moved